src/package/nuspec/TestPlatform.TestHost.nuspec

```diff
@@ -40,45 +40,35 @@
     <file src="_._" target="lib\net45\" />
 
     <!-- netcoreapp1.0 -->
-    <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\Microsoft.TestPlatform.CommunicationUtilities.dll" target="lib\netcoreapp1.0\" />
-    <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\Microsoft.TestPlatform.CrossPlatEngine.dll" target="lib\netcoreapp1.0\" />
-    <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\Microsoft.VisualStudio.TestPlatform.Common.dll" target="lib\netcoreapp1.0\" />
-    <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\Microsoft.TestPlatform.Utilities.dll" target="lib\netcoreapp1.0\" />
+    <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\Microsoft.VisualStudio.TestPlatform.*.dll" target="lib\netcoreapp1.0\" />
+    <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\Microsoft.TestPlatform.*.dll" target="lib\netcoreapp1.0\" />
     <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\testhost.dll" target="lib\netcoreapp1.0\" />
     <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\testhost.deps.json" target="lib\netcoreapp1.0\" />
     <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\x86\msdia140.dll" target="lib\netcoreapp1.0\x86\" />
     <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\x64\msdia140.dll" target="lib\netcoreapp1.0\x64\" />
-    
-    <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\win7-x64\testhost.dll" target="build\netcoreapp1.0\x64" />
-    <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\win7-x64\testhost.exe" target="build\netcoreapp1.0\x64" />
-    <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\win7-x86\testhost.x86.dll" target="build\netcoreapp1.0\x86\" />
-    <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\win7-x86\testhost.x86.exe" target="build\netcoreapp1.0\x86\" />
+  
+    <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\win7-x64\*" target="build\netcoreapp1.0\x64\" />
```

Comment:
Didn't you say this would make the package huge?

Reply:
No, that folder only contains one more file, not the entire runtime.

Reply:
Why do we need testhost.deps.json? It won't be used, because we provide deps.json of the tested project.

Reply:
We actually don't need it, I updated package the nuspec.